pkg/cachemanager/cachemanager.go

```diff
@@ -440,3 +433,7 @@ func (c *CacheManager) wipeCacheIfNeeded(ctx context.Context) {
 		c.needToList = true
 	}
 }
+
+func (c *CacheManager) GetSyncMetricsCache() *syncutil.MetricsCache {
```

Comment:
We should not be exporting this private variable. This file should have no delta.